collector/chassis_collector.go

```diff
@@ -302,19 +305,18 @@ func (c *ChassisCollector) Collect(ch chan<- prometheus.Metric) {
 			// process NetapAdapter
 
 			networkAdapters, err := chassis.NetworkAdapters()
-
 			if err != nil {
-				log.Infof("Errors Getting NetworkAdapters from chassis : %s", err)
+				chassisLogContext.WithField("operation", "chassis.NetworkAdapters()").WithError(err)
```

Comment:
```suggestion
				chassisLogContext.WithField("operation", "chassis.NetworkAdapters()").WithError(err).Error("error while querying network adapters")
```
You need to provide a log message with the `.Error()`, `.Info()`, or equivalent method, otherwise you'll get no logging output by just using the `.WithXXX()` methods :D